### PR TITLE
Handle 360_CUBE projection

### PR DESCRIPTION
**Pull request by dillontiner**

### CODE REVIEW

**Comment by jbroberg** [comment]
Awesome. Do you have a link to a good sample we can test with? I have some of my own but I'd love to see what you tested against.

**Comment by dillontiner** [comment]
Added the cubemap video I've been using for reference. It's just a [sample](https://vimeo.com/214403336) I downloaded and transformed with [transform360](https://github.com/facebook/transform360). The video itself leaves some visible seams on the cube edges for reasons explored in this [blog post](https://medium.com/@sneakymax/rendering-a-cube-mapped-360-video-in-three-js-d6eca25a3d76), which fall on the encoding side.

**Comment by dillontiner** [comment]
I paired each projection's `this.movieMaterial` assignment with its `this.movieGeometry` definition.

I also updated the cubemap faces so they precisely match the equivalent equirectangular video. Just for reference, the cubemap I worked with has the following orientation from the output of transform360:
|    Left    |   Right   |    Top    |
| Bottom |   Front   |   Back   |

File: src/plugin.js [hunk 1]
```diff
@@ -156,44 +160,48 @@ class VR extends Plugin {
...
-      const face3 = [new THREE.Vector2(0.666, 0.5), new THREE.Vector2(1, 0.5), new THREE.Vector2(1, 1), new THREE.Vector2(0.666, 1)];
-      const face4 = [new THREE.Vector2(0, 0), new THREE.Vector2(0.333, 1), new THREE.Vector2(0.333, 0.5), new THREE.Vector2(0, 0.5)];
-      const face5 = [new THREE.Vector2(0.333, 1), new THREE.Vector2(0.666, 1), new THREE.Vector2(0.666, 0.5), new THREE.Vector2(0.333, 0.5)];
-      const face6 = [new THREE.Vector2(0.666, 1), new THREE.Vector2(1, 0), new THREE.Vector2(1, 0.5), new THREE.Vector2(0.666, 0.5)];
+      this.movieGeometry = new THREE.BoxGeometry(256, 256, 256);
+      this.movieMaterial = new THREE.MeshBasicMaterial({ map: this.videoTexture, overdraw: true, side: THREE.BackSide });
+
+      const left = [new THREE.Vector2(0, 0.5), new THREE.Vector2(0.333, 0.5), new THREE.Vector2(0.333, 1), new THREE.Vector2(0, 1)];
+      const right = [new THREE.Vector2(0.333, 0.5), new THREE.Vector2(0.666, 0.5), new THREE.Vector2(0.666, 1), new THREE.Vector2(0.333, 1)];
+      const top = [new THREE.Vector2(0.666, 0.5), new THREE.Vector2(1, 0.5), new THREE.Vector2(1, 1), new THREE.Vector2(0.666, 1)];
```
Comment (BrandonOCasey) [inline src/plugin.js]: 👍

**Comment by ishita12** [comment]
QA LGTM

**Pull request merged**